MODULE.md

```diff
@@ -0,0 +1,262 @@
+
+##### Remove Pending Validator (remove-pending):
+
+- Syntax: `manifestd tx poa remove-pending [validator]`
+
+  - Parameters:
+
+    - `validator`: The validator's operator address.
+
+    **Example:** `manifestd tx poa remove-pending manifestvaloper1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct`
+
+##### Remove Validator (remove):
+
+- Syntax: `manifestd tx poa remove [validator]`
+
+  - Parameters:
+
+    - `validator`: The validator's operator address.
+
+    **Example:** `manifestd tx poa remove manifestvaloper1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct`
+
+## Token Factory Module
+
+The Token Factory module as it is implemented on the Manifest Network, allows the PoA admin to have granular control over the creation and management of tokens on the Manifest Network. The admin can mint, burn, edit, and transfer tokens to other accounts from any account.
+
+### Module Functionality
+
+#### Token Minting:
+
+- Create a token with a specific denom
+- Mint a token with a specific amount and denom to your account
+- Mint a token with a specific amount and denom to another account
+
+#### Token Burning:
+
+- Burn a token with a specific amount and denom from your account
+- Burn a token with a specific amount and denom from another account
+
+#### Token Administration:
+
+- Change the admin address for a factory-created denom
+- Force transfer tokens from one address to another address
+
+#### Metadata Management:
+
+- Change the base metadata for a factory-created denom
+
+#### Commands
+
+##### Burn (burn):
+
+- Syntax: `manifestd tx token-factory burn [amount]`
+
+  - Parameters:
+    - `amount`: The amount and denom of the token you would like to burn from your account.
+
+  **Example:** `manifestd tx tokenfactory burn 1<denom>`
+
+##### Burn From (burn-from):
+
+- Syntax: `manifestd tx token-factory burn-from [address] [amount]`
+
+  - Parameters:
+    - `address`: The address of the account you would like to burn the tokens from.
+    - `amount`: The amount and denom of the token you would like to burn.
+
+  **Example:** `manifestd tx tokenfactory burn-from manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct 1<denom>`
+
+##### Mint (mint):
+
+- Syntax: `manifestd tx token-factory mint [amount]`
+
+  - Parameters:
+    - `amount`: The amount and denom of the token you would like to mint to your account.
+
+  **Example:** `manifestd tx tokenfactory mint 1<denom>`
+
+##### Change Admin (change-admin):
+
+- Syntax: `manifestd tx token-factory change-admin [denom] [new-admin-address]`
+
+  - Parameters:
+    - `denom`: The denom of the token that you would like to change the admin for.
+    - `new-admin-address`: The new admin's wallet address.
+
+  **Example:** `manifestd tx tokenfactory change-admin <denom> manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct`
+
+##### Create Denom (create-denom):
+
+- Syntax: `manifestd tx token-factory create-denom [subdenom]`
```

Comment:
I beleive the token factory flow is as follows 

- create-denom: This creates a token. 
- modify-metadata: This adds information that is otherwise null
- mint or burn
- mint or burn to and from

Reply:
Interesting. Why the two-step process? Does it make sense to create a token with null parameters?

Reply:
yes, a token with null params is perfectly fine to do. It's not a hard requirement. So if your token needs the params, you set them. If not, it's fine as is.

A UI or multisig can do these 2 in 1 signature. But this is just CLI things